actions/init-ci/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Init CI
+description: Sets up common environment for CI jobs
+
+inputs:
+  buildx:
+    required: false
+    description: Enable buildx system (needed just to enable some docker build-push-action features)
+  gcp_service_account:
+    required: true
+    description: Login to gcloud and gcr with this service account (needed in order to push images and run gcloud)
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+      if: ${{ inputs.buildx }}
+    - name: Authenticate to Google Cloud
+      if: ${{ inputs.gcp_service_account }}
```

Comment:
I think that you can remove the `if`s here since it's a required field here

Reply:
it shouldn't be, 10x for noticing :)

Reply:
Just doin' my job ;)